src/components/dialogs/network-modifications/static-var-compensator/creation/susceptance-area.tsx

```diff
@@ -0,0 +1,154 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import Grid from '@mui/material/Grid';
+import {
+    B0,
+    MAX_Q_AT_NOMINAL_V,
+    MAX_SUSCEPTANCE,
+    MIN_Q_AT_NOMINAL_V,
+    MIN_SUSCEPTANCE,
+    Q0,
+    SLIDER_Q_NOMINAL,
+    SLIDER_SUSCEPTANCE,
+} from 'components/utils/field-constants';
+import { FloatInput, SliderInput, TextInput } from '@gridsuite/commons-ui';
+import { gridItem, ReactivePowerAdornment, SusceptanceAdornment } from '../../../dialogUtils';
+import React, { FunctionComponent, useEffect } from 'react';
+import { useFormContext } from 'react-hook-form';
+import { getFloatNumber } from './stand-by-automaton-form-utils';
+import { isValidPercentage } from '../../../percentage-area/percentage-area-utils';
+
+export interface SusceptanceAreaProps {
+    isSusceptance: boolean;
+    isQFixe: boolean;
+    min: number;
+    max: number;
+}
+export const SusceptanceArea: FunctionComponent<SusceptanceAreaProps> = ({ isSusceptance, isQFixe, min, max }) => {
```

Comment:
In my opinion, we can make the component more generic with 4 JSX sub-component, min (TextField), max (TextField), slider(SliderInput), input(TextInput) with props {minLabel, maxLabel, sliderLabel, inputLabel, sliderName, inputName, min, max}

Reply:
new solution is useWatch to be reactive with changes from setpoint form